client/src/components/questions_and_answers/Search.jsx

```diff
@@ -1,21 +1,34 @@
 import React from 'react';
 
-const search = ({ setQuestions, questions }) => {
-  const storage = questions;
-  const filterResults = (text) => {
+const { useState, useEffect } = React;
+
+
+const search = ({ questions, filter, setFilter }) => {
+
+  const [searchInput, setSearchInput] = useState('');
+  const [filteredList, setFilteredList] = useState('');
+
+  const handleChange = () => {
     // As soon as entry hits 3 chars, Begin filtering the List of questions
     // To include on relevant questions
-    if (text.length > 3) {
-      questions
+    if (searchInput.length > 2) {
+      setFilter(questions.filter((question) => question.question_body.match(searchInput)));
```

Comment:
Although this was a good one liner, you could split this one up to lessen complexity and clarity.